ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -56,7 +56,10 @@ spec:
           phc2sysOpts: "-a -r -n 24"
     - fileName: SriovOperatorConfig.yaml
       policyName: "config-policy"
+      complianceType: mustonlyhave
```

Comment:
A question on the "mustonlyhave" part of this. Is it required to set the proper role for new clusters, or was it added to handle existing clusters which may have a conflicting role (eg master)? 
In the past some configuration (I believe OperatorGroup) caused issues with high CPU use when we used "mustonlyhave" as the compliance type due to a battle between multiple controllers setting annotations on the CR. Can we leave this as "complianceType: musthave" and add a note that existing clusters need to use mustonlyhave to remove the old annotation and then switch to musthave?